presidio-analyzer/analyzer/analyzer_engine.py

```diff
@@ -120,7 +134,10 @@ def analyze(self, text, entities, language, all_fields):
             if current_results:
                 results.extend(current_results)
 
-        return AnalyzerEngine.__remove_duplicates(results)
+        results = AnalyzerEngine.__remove_duplicates(results)
+        self.app_tracer.trace(request_id, repr(results))
```

Comment:
Same here. it's better to avoid `repr` if we want to make sure the entire object is serialized